### ISSUE TITLE
Great job!

### ISSUE THREAD

**Issue by GamebP** [KetamineAddict/freedom#1]
I really like the things you made and improved the original Freedom. 
Are you going to make like more features to it or stick as-is?

Could be possible to make a magnet aimbot (for cursor) - if I'm not saying in harsh way. 🤔

**Comment by KetamineAddict** [comment]
I will see what i can do, currently im working on the distance logic. After that I will implement some other features to the aimbot and relax. Currently dont have that much time so thats why the updates came to a halt but I will continue in a week or so. Stay tuned ;D 

And thanks for the feedback<3

**Comment by GamebP** [comment]
Ah cool!! Looking for the new updates to come out then 👍 
Do you have a discord or somenthing? Maybe I could have some more ideas about current things that original Freedom dosent have.

**Comment by GamebP** [comment]
@KetamineAddict

**Comment by KetamineAddict** [comment]
Just post them here. I dont use discord for work related content.

**Comment by GamebP** [comment]
Make relax have a option to make  
x amount of misses or
x amount of 50 or    
x amount of 100 or  
x amount of 300

**Comment by KetamineAddict** [comment]
I could make it so that it randomly presses a different or no button at all with a % chance. Not fixed x values. 
Its on my todo list now. Thanks for the suggestion.

**Comment by KetamineAddict** [comment]
The rest like 
amount of 50s 100s and 300s have to be roughly estimated based on the parameters you have.

For example you could lower the amount of "random extremity MIN" & "MAX" so that it hits more often outside of the normal distribution. The default values I set just work best with the kind of maps I play (7*-9*) and if it gets more difficult I just put the sliders up more to make it seem more legit.

I have a lot planned already and in the end you will be glad with the public version.

I will make a private version that is way more advanced and I want a driver for it which I will potentially let Microsoft sign. (I need money for the driver so I will probably sell the private version some day)

Not everything set in stone already so this is subject to change.

But this will become the best OSU bot/assistant/cheat... whatever you wanna call it. 
Just my first project and a love of mine I want to show my appreciation for.

**Comment by GamebP** [comment]
Yep. It is a better improvment - with more features in relax and aimbot.

**Comment by GamebP** [comment]
What about the aimbot that slowly moves to the circle/slider as it goes away (gives a miss if not pressed) 
And when it will fade away it goes fast, 
Could be [Ease](https://cubic-bezier.com/#.6,.1,.8,.8)

**Comment by KetamineAddict** [comment]
I will rather do more bezier variations like the 6 point one i created here. 
The 4 point one you suggested is also good and i will include it into the calculation. 
Will be % of 3, 4 and 6 point. maybe even more in the future.

**Comment by GamebP** [comment]
could be customizable in near future right?

**Comment by KetamineAddict** [comment]
yes

**Comment by GamebP** [comment]
Anything new?

**Comment by KetamineAddict** [comment]
Not yet, been busy irl and only working on other private stuff. In 2 weeks I will resume work.

**Comment by GamebP** [comment]
Alright

KetamineAddict ***@***.***> (šajā datumā: pirmd., 2024. g. 1.
jūl. 00:50) rakstīja:

**Comment by GamebP** [comment]
https://youtu.be/SaH6qOVrlAc Here I have somenthing to request about cursor movment like how it moves slow/fast at [01:10](https://youtu.be/SaH6qOVrlAc?si=f0fFzfbVM87bxTzN&t=70) it shows when the circle hit circle goes away the cursor goes faster. 
This could be implemented in the different Aimbot Algorithm's that are currently.
